src/index.js

```diff
@@ -12,7 +12,15 @@ import { withinRange } from './withinRange';
 
 // do not export anything else here to keep window.eventDrops as a function
 export default ({ d3 = window.d3, ...customConfiguration }) => {
-    const chart = selection => {
+    const onResize = callback => {
```

Comment:
No need for these functions. Just keep a reference to the event listener to be able to remove it in the `destroy` method.